test/system/070-build.bats

```diff
@@ -698,7 +699,7 @@ EOF
     # we're happy.
     if ! is_remote; then
         is "$output" \
-           ".* pull policy is .never. but .* could not be found locally" \
+           "Error: error creating build container: quay.io/libpod/nosuchimage:nosuchtag: image not known" \
```

Comment:
This is causing the rootless CI failures. And as I look at it closely, I think this is a regression and a bug: this test should never emit "image not known" as a failure. You should not be touching this line. If you are touching this line, it means your PR is breaking `--pull-never`.

Reply:
Thanks for checking, will address it now.  It's not a bug but the error message has changed (everything behind `podman image *` is new).  If you want to keep the error message, please let me know and I will change it with the next iteration of libimage.

Reply:
"image not known" is ambiguous: I can't tell if it means "I do not have that image locally, and cannot check because of --pull-never", or "I do not have that image locally, and I went and checked anyway, and did not find it in the registry".